security/Security.php

```diff
@@ -1123,6 +1124,18 @@ public static function logout_url() {
 		return self::config()->logout_url;
 	}
 
+	/**
+	 * Get the URL of the lost password page
+	 *
+	 * @return string
+	 */
+	public static function lost_password_url()
+	{
+		$selfControllerClass = __CLASS__;
+		$selfController      = new $selfControllerClass();
```

Comment:
I agree, I actually had that originally bit switched it out for the same method that has been used elsewhere in the file to keep things consistent (https://github.com/silverstripe/silverstripe-framework/blob/ff50b441393ef4950972891dc5b7fee7a9453395/security/Security.php#L656).

I had considered updating both of them but decided against it to keep this pull request focussed.